doc/content/examples/example5b.md

```diff
@@ -0,0 +1,69 @@
+# Example 5b: Dynamic response of a simplified 1D model of Nuclear Power Plant  with mesh generated using BeamMeshGenerator.
+
+## Model
+
+This example demonstrates the use of [BeamMeshGenerator](source/index.md) to create a 1D line model of the simplified stick model of a nuclear power plant taken from the SASSI2000 user manual [!citet](sassiuser2006), described in [Example 5a](examples/example5a). The rotations of the stick model is restrained at the base and the same input acceleration time history as shown in [Example 5a](examples/example5a) is applied at the base of the structure. The  dynamic responses thus obtained are compared with the dynamic responses obtained from input file which imports a mesh file generated using [Cubit](https://cubit.sandia.gov/).
```

Comment:
Mention that the basemat is not modeled. 

'The rotations of the stick model are restrained at the base ...'

Extra space in 'The dynamic responses thus obtained ..'